packages/base/src/dialogs/symbology/vector_layer/VectorRendering.tsx

```diff
@@ -78,6 +80,7 @@ const VectorRendering = ({
             okSignalPromise={okSignalPromise}
             cancel={cancel}
             layerId={layerId}
+            SymbologyTab={SymbologyTab}
```

Comment:
```suggestion
            symbologyTab={symbologyTab}
```

And similarly with the property names

Reply:
482f458